networking_oneview/conf/oneview.py

```diff
@@ -38,9 +38,10 @@
     cfg.IntOpt('ov_refresh_interval',
                default=3600,
                help='Interval between periodic task executions in seconds.'),
-    cfg.BoolOpt('developer_mode',
+    cfg.BoolOpt('force_sync_delete_ops',
                 default=False,
-                help='Only set this option as true if under developer mode.')
+                help=('Set this option as true allow sync ro delete outdated ',
+                      'network and connections'))
```

Comment:
If set to true, allows Networking OneView Synchronization delete outdated network and connections.

Reply:
Agreed... this message needs rewriting.